README.md

```diff
@@ -169,6 +170,30 @@ window.ai.addEventListener((event: EventType, data: unknown) => {
 })
 ```
 
+**(BETA) Generate 3D Objects**: generate a 3D object using a specified model or the user-preferred model.
+
+```ts
+window.ai.BETA_generate3DObject(
+    input: PromptInput,
+    options?: ThreeDOptions
+): Promise<MediaOutput[]>
+```
+
+The `BETA_generate3DObject` function allows you to generate 3D objects with a defined model and options. The input should be a `PromptInput` `{ prompt : string }`. The options parameter is optional and accepts `ThreeDOptions` customize the media generation request.
+
+Here's an example request:
+
+```javascript
+const result = await window.ai.BETA_generate3DObject(
+  { "prompt": "a glazed donut" }, 
+  { "numInferenceSteps": 32, "mimeType": "application/x-ply" });
```

Comment:
same note here re: mimeType (remove for now since it's hardcoded to ply)

Reply:
removed